src/components/submission/decrypt.vue

```diff
@@ -277,11 +277,9 @@ export default {
     download(event) {
       const a = event.target.closest('a');
       if (a == null) return;
-      const willDownload = !a.classList.contains('disabled') &&
+      const willDownload = this.managedKey == null ||
         this.$refs.form.reportValidity();
-      if (this.managedKey == null) {
-        if (!willDownload) event.preventDefault();
-      } else {
+      if (this.managedKey != null) {
         event.preventDefault();
         if (willDownload) this.decrypt(a.getAttribute('href'));
       }
```

Comment:
Because of `useDisabled()`, it's now guaranteed that `a` doesn't have the `disabled` class. That means that if `this.managedKey == null`, we never need to prevent default now: there's never a reason to stop the download now. (If `this.managedKey != null`, we always prevent default, because we initiate the download another way, within `this.decrypt()`.) Note that if `this.managedKey == null`, we don't need to check form validity, because there's no decryption passphrase input to validate.